ang/civicase/CaseListTable.html

```diff
@@ -1,5 +1,5 @@
-<table ng-show="!caseIsFocused" class="case-list-table">
-  <thead calculate-scroll-width>
+<table ng-show="!caseIsFocused" class="table civicase__table--caselist">
```

Comment:
Case list is a component, and its name should be `civicase__case-list`. That means we dont need `civicase__table--caselist"` class

Reply:
Done

Reply:
It now uses `civicase__list-table`

Reply:
Thanks for changing, but the naming convention is still wrong. Its not any table, its case-list table. So `civicase__case-list-table`

Reply:
Other names like `civicase__list-table__header` should also be changed to `civicase__case-list-table__header`